pkg/operator/prerequisites_test.go

```
@@ -0,0 +1,150 @@
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			origDelay := delayIteration
+			origIterations := numIterations
+			delayIteration = 0
+			numIterations = 3
+			defer func() {
+				delayIteration = origDelay
+				numIterations = origIterations
+			}()
+
+			kubeClient := fakecore.NewSimpleClientset()
```

golangci-lint (2.12.2)

[error] SA1019: fakecore.NewSimpleClientset is deprecated: NewClientset replaces this with support for field management, which significantly improves server side apply testing. NewClientset is only available when apply configurations are generated (e.g. via --with-applyconfig). 

(staticcheck)